packages/paste-website/src/components/svg/github-icon.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import {useUID} from 'react-uid';
+
+export interface GithubIconProps {
```

Comment:
This filename should be `GithubIcon.tsx` because it's a React component and to match TwilioIcon 🙏

Reply:
Done